lib/better_errors/templates/main.erb

```diff
@@ -915,6 +939,21 @@
             this.onNavigateHistory(1);
             ev.preventDefault();
             return false;
+        } else if (ev.keyCode == 76 && ev.ctrlKey) { // ^L
+            ev.preventDefault();
+            this.clearScreen();
+        } else if (ev.keyCode == 65 && ev.ctrlKey) { // ^A
+            ev.preventDefault();
+            this.beginningOfLine();
+        } else if (ev.keyCode == 69 && ev.ctrlKey) { // ^E
+            ev.preventDefault();
+            this.endOfLine();
+        } else if (ev.keyCode == 75 && ev.ctrlKey) { // ^K
+            ev.preventDefault();
+            this.killLine()
+        } else if (ev.keyCode == 67 && ev.ctrlKey) { // ^C
```

Comment:
Won't this interfere with Linux/OS X users trying to copy sections of text from the input box?